clover.rb

```diff
@@ -138,7 +149,7 @@ def self.freeze
 
   plugin :sessions,
     key: "_Clover.session",
-    cookie_options: {secure: !%w[test development].include?(ENV["RACK_ENV"])},
+    cookie_options: {secure: Config.production?},
```

Comment:
Although it seems unlikely to be a cause of error, positively identify the exceptions for secure mode (development, test) like the original code. A typo of `RACK_ENV` is in the failure mode somewhere.

Reply:
Something like your proposal.